packages/hub/src/session.ts

```diff
@@ -66,7 +66,11 @@ export class Session implements SessionInterface {
     if (typeof context.duration === 'number') {
       this.duration = context.duration;
     } else {
-      this.duration = this.timestamp - this.started;
+      // Otherwise:
+      // hub.startSession();
+      // hub.captureSession();
+      // can produce either 0 or 1, depending on the mood
+      this.duration = Math.floor(this.timestamp - this.started);
```

Comment:
https://develop.sentry.dev/sdk/sessions/#session-update-payload documents `duration` to be a `float`

What exactly was the problem here? If float is not being ingested correctly then we might need changes elsewhere.

Reply:
‘Date.now() - Date.now()’ which is what we have here will always produce int.

Reply:
If both are ints, what's the deal with rounding? This doesn't make sense.

If we want to always report zero then Math.floor is not part of the solution.

Reply:
New session is initially at 0, but captureSession is passing its own Date.now(). Its possible that 2 calls followed after each other (as in comments) will have 2 differents Date.now (by 1ms), which will produce a duration of 1.

Reply:
Team talked about this on a call, decided that we:

1. Do not send duration for JS Browser sessions.
2. Fix duration calculation to report seconds instead of milliseconds and to use the same clock as we use for errors and transactions. See [`timestampInSeconds`](https://github.com/getsentry/sentry-javascript/blob/55f85b0f4617de6c49a7dfa1b9adca1b95b4b272/packages/utils/src/time.ts#L119).
3. For this PR, revert the above changes to `packages/hub/src/session.ts` and merge the changes to `packages/browser/src/sdk.ts`.